examples/siro_sandbox/controllers/gui_controller.py

```diff
@@ -195,9 +197,21 @@ def __init__(
         self._hint_distance_multiplier = None
         self._hint_grasp_obj_idx = None
         self._hint_drop_pos = None
+        self._hint_throw_vel = None
+        self._hint_reach_pos = None
+        self._is_picking = None
+        self._iter_pose = 0
         self._cam_yaw = 0
         self._saved_object_rotation = None
         self._recorder = recorder
+        self._obj_to_grasp = None
+        self._thrown_object_collision_group = CollisionGroups.UserGroup7
+        self._last_object_thrown_info = None
+
+        self.ind = 0
```

Comment:
Agreed! Making suggestions here, since it seems I cannot create a suggested change for all the lines:
```
self._hint_throw_vel_vec = None # The velocity vector at which the object will be thrown 
self._hint_reach_pos = None # The location at which to pick the object. If None, no object should be picked
self._is_picking_reach_pos = None # Same as _hint_reach_pos but is set to None as soon as the humanoid reaches the object
self._iter_hand_pose = 0 # Iterator to compute the hand position when reaching objects. 
```